django/core/paginator.py

```diff
@@ -51,10 +51,7 @@ def validate_number(self, number):
         if number < 1:
             raise EmptyPage(_("That page number is less than 1"))
         if number > self.num_pages:
-            if number == 1 and self.allow_empty_first_page:
```

Comment:
I think this case can never be possible, because if `number == 1`, `self.num_pages` has to be `0` and that can't be possible as long `allow_empty_first_page` is `True`

Reply:
Good catch :+1: